components/collective-page/sections/Contribute.js

```diff
@@ -164,26 +185,17 @@ class SectionContribute extends React.PureComponent {
                 </ContainerSectionContent>
 
                 <ContributeCardsContainer ref={ref}>
-                  <Box px={CONTRIBUTE_CARD_PADDING_X}>
-                    <ContributeCustom
-                      collective={collective}
-                      contributors={financialContributorsWithoutTier}
-                      stats={contributorsStats}
-                      hideContributors={hasNoContributor}
-                    />
-                  </Box>
-                  {sortedTiers.map(tier => (
-                    <Box key={tier.id} px={CONTRIBUTE_CARD_PADDING_X}>
-                      <ContributeTier collective={collective} tier={tier} hideContributors={hasNoContributor} />
-                    </Box>
-                  ))}
-                  {isAdmin && (
-                    <Box px={CONTRIBUTE_CARD_PADDING_X}>
-                      <CreateNew data-cy="create-contribute-tier" route={createContributionTierRoute}>
-                        <FormattedMessage id="Contribute.CreateTier" defaultMessage="Create Contribution Tier" />
-                      </CreateNew>
-                    </Box>
-                  )}
+                  <ContributeTiersPanel
+                    isAdmin={isAdmin}
+                    collective={collective}
+                    sortedTiers={sortedTiers}
+                    hasNoContributor={hasNoContributor}
+                    contributorsStats={contributorsStats}
+                    handleSettingsUpdate={handleSettingsUpdate}
+                    CONTRIBUTE_CARD_PADDING_X={CONTRIBUTE_CARD_PADDING_X}
```

Comment:
Let's keep parameters names camelCase. Uppercase is for constants.

```suggestion
                    contributeCardPaddingX={CONTRIBUTE_CARD_PADDING_X}
```